### PR TITLE
Grunt tasks

### PR DESCRIPTION
**Pull request by MattSurabian**
Add grunt tasks so it's easy for anyone to run `jscs`, `jshint`, and `tap` checks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "main": "index.js",
   "scripts": {
-    "test": "node test.js | tspec"
+    "test": "grunt && node test.js | tspec"
```
Comment (calvinmetcalf) [inline package.json]: "test": "jscs index.js test.js examples/ && node test.js | tspec"

then no need for grunt (though jscs would now need to be a dev dep)

**Comment by calvinmetcalf** [comment]
why both jscs and jshint ?

**Comment by MattSurabian** [comment]
jscs ensures consistent code style and jshint ensures js best practices. Both should now be added as dev dependencies and everything is just running through grunt

**Comment by calvinmetcalf** [comment]
this entire thing can be replaced by changing the test line to

``` json
"test": "npm run lint && node test.js | tspec"
"lint": "jscs index.js test.js examples/ && jshint index.js test.js examples/"
```

and using jscs and jshint directly instead of the grunt plugins able to remove the tasks and all that jazz

**Comment by MattSurabian** [comment]
Yeah I guess since we're not doing any more complicated build steps we don't need all dat business.

**Comment by MattSurabian** [comment]
closed in favor of #8

**Pull request closed without merging**